src/app/contextBroker/contextBroker.cpp

```diff
@@ -63,7 +63,7 @@
 #include <openssl/ssl.h>
 #include <string>
 #include <vector>
-#include <limits.h>
+#include <limits.h>                            
```

Comment:
```suggestion
#include <limits.h>
```

To avoid extra trailing whitespace